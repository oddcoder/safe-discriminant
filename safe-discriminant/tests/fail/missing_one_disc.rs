use safe_discriminant::Discriminant;

#[derive(Discriminant)]
#[repr(u8)]
pub enum Foo {
    A = 0,
    B,
}

fn main() {}
