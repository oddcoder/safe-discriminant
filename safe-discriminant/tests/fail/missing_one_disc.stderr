error: Missing explicit discriminant. Note: If you want to reason about discriminants, but do not care about concrete values, consider using `core::mem::Discriminant` instead.
 --> tests/fail/missing_one_disc.rs:7:5
  |
7 |     B,
  |     ^
